static/usage/select/customization/button-text/index.md

```diff
@@ -0,0 +1,12 @@
+import Playground from '@site/src/components/global/Playground';
+
+import javascript from './javascript.md';
+import react from './react.md';
+import vue from './vue.md';
+import angular from './angular.md';
+
+<Playground
+  size="300px"
```

Comment:
Can we make this a bit bigger so the MD alert does not need to scroll: 
![image](https://user-images.githubusercontent.com/2721089/166535532-61998ce3-1562-4b24-b84e-d4a234416abf.png)